docs/guide/usage-widgets.md

```diff
@@ -19,15 +19,14 @@ framework features. All widgets belong to `\yii\bootstrap` namespace:
 - [[yii\bootstrap\Tabs|Tabs]]
 
 
-## Customize widget CSS classes <span id="widgets-customize-css-classes"></span>
+## Customizing widget CSS classes <span id="customizing-css-classes"></span>
 
-Widgets allow quick composition of the HTML for particular Bootstrap components, which requires usage of the
-Bootstrap CSS classes. Those classes are added automatically by the widgets. However usually widget provides
-fields or properties, which allow customization of the CSS styles for particular HTML tags.
+These widgets allow quick composition of the HTML for the Bootstrap components that require the use of the Bootstrap CSS classes.
```

Comment:
Since this is a beginning of the sub-topic using conjunction 'these' should be avioded. Better be "Bootstrap widgets", "Widgets" or similar.